### ISSUE TITLE
path option arguments don't work correctly

### ISSUE THREAD

**Issue by chebum** [blakeembrey/metalsmith-pagination#19]
It looks like there is an issue in the interpolate method. For example, this path setting won't be replaced correctly: 
path: 'blog/page:num_en.html'

After performing the interpolate method, it will return blog/pageundefined.html

**Comment by carolineBda** [comment]
It seems that the pagination doesn't work with metalsmith-collections@0.9.0
Is this plugin still maintain?

**Comment by blakeembrey** [comment]
Yes, it does. Yes, it is. And your question seems unrelated to the issue you commented on.

**Comment by carolineBda** [comment]
Sorry I should have been more precise. When you upgrade metalsmith-collections from 0.7.0 to 0.9.0 then the path field in the file object listed in **pagination.files** become a string 
`"articles/le-theatre-entreprise.md" `

instead of being the usual object :
```
{ root: '',
  dir: 'articles',
  base: 'le-theatre-entreprise.md',
  ext: '.md',
  name: 'lexique-satisfaction-client-W',
  href: '/articles/le-theatre-entreprise.md,
  dhref: '/articles/' 
}
```

I thought it was related to the issue above but if it isn't I will open a new issue.

**Comment by blakeembrey** [comment]
No, the issue above is just about interpolation not working because of how word matching works with the syntax. There's a related PR that unfortunately wasn't linked to this issue or completed. 

As for the issue you have, go ahead and create a new one. I'll take a look at it, seems odd - I didn't encounter this before. Is it a very new release? What's your configuration look like?

**Comment by carolineBda** [comment]
Done : https://github.com/blakeembrey/metalsmith-pagination/issues/30
